TS.fsx

```diff
@@ -465,17 +465,33 @@ let EmitStaticInterface flavor (i:Browser.Interface) =
     // Some types are static types with non-static members. For example, 
     // NodeFilter is a static method itself, however it has an "acceptNode" method
     // that expects the user to implement. 
-    let hasNonStaticMember = 
-        match i.Methods with
-        | Some ms -> ms.Methods |> Array.exists (fun m -> m.Static.IsNone)
-        | _ -> false
+    let hasNonStaticMember =
+        let hasNonStaticMethod =
+            match JsonItems.getAddedItemsByInterfaceName ItemKind.Method flavor i.Name with
+            | [||] ->
+                match i.Methods with
+                | Some ms -> ms.Methods |> Array.exists (fun m -> m.Static.IsNone)
```

Comment:
The original idea was the there is no static property, the only thing similar may be a static constant, which should be added as `constant` in the json file because it has a value. Though you may be right that it is better checking here to be conservative.